kuma/static/js/lang-switcher.js

```diff
@@ -3,7 +3,19 @@
     var neverShowNoticeKey = 'never-show-locale-notice';
     var neverShowNotice = getNeverShowNotice();
     function storeLocaleChange(code, name) {
-        sessionStorage.setItem(sessionStorageKey, JSON.stringify({code: code, name: name}));
+        if (!isExistingLocaleCookieExist(code)){
+            sessionStorage.setItem(sessionStorageKey, JSON.stringify({code: code, name: name}));
+        }
+    }
+
+    function getCookie(name) {
+	    var v = document.cookie.match('(^|;) ?' + name + '=([^;]*)(;|$)');
+	    return v ? v[2] : null;
+	}
+
+    function isExistingLocaleCookieExist(code) {
+        var cookieName = document.querySelector('meta[name="lang-cookie"]').content;
```

Comment:
I wish we could inject this name directly into the JS, but we'll probably need to improve our asset pipeline first.

I'd prefer [jinja2/includes/config.html](https://github.com/mozilla/kuma/blob/master/jinja2/includes/config.html), where we already pass Django settings into JS land.